### PR TITLE
SORQA-307

### PR DESCRIPTION
**Pull request by lb-sgent**
image

### CODE REVIEW

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/CreateNewEventSteps.java [hunk 1]
```diff
@@ -252,6 +254,23 @@ public CreateNewEventSteps(
...
+        (String eventIdentificationSource) -> {
+          newEvent = collectEventUuid();
+          String timestamp = String.valueOf(System.currentTimeMillis());
+          webDriverHelpers.fillInWebElement(TITLE_INPUT, "EVENT_AUTOMATION" + timestamp);
+          selectEventIdentificationSource(eventIdentificationSource);
+          selectResponsibleRegion(RegionsValues.VoreingestellteBundeslander.getName());
+          selectResponsibleDistrict(DistrictsValues.VoreingestellterLandkreis.getName());
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(NEW_EVENT_CREATED_MESSAGE);
+          webDriverHelpers.clickOnWebElementBySelector(EVENT_TAB);
```
Comment (mp-sgent) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/CreateNewEventSteps.java:269]: Navigation to event tab after event creation should be already available as a separate step, so I suggest removing it from this step and using already defined for that action.

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/CreateNewEventSteps.java [hunk 2]
```diff
@@ -252,6 +254,23 @@ public CreateNewEventSteps(
...
+          Assert.assertTrue(
```
Comment (mp-sgent) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/CreateNewEventSteps.java:270]: This assert should be treated as a separate step and be placed in EditEventSteps cause action that it performs in in that directory.

File: sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature [hunk 3]
```diff
@@ -820,4 +820,17 @@ Feature: Create events
...
-    And I check that Nickname is not visible in Person search popup
\ No newline at end of file
+    And I check that Nickname is not visible in Person search popup
+
+  @issue=SORDEV-7094 @env_main
+  Scenario: Test Event identification source fields
+    Given I log in with National User
+    And I click on the Events button from navbar
+    And I click on the NEW EVENT button
+    And I create a new event with event identification source UNKNOWN
```
Comment (mp-sgent) [inline sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature:830]: Test suite can be paramatrized with that event identification source value.

**Review by mp-sgent — CHANGES_REQUESTED** [review]

**Review by rdutu-vg — APPROVED** [review]

**Comment by lb-sgent** [comment]
image

**Review by hms-sgent — APPROVED** [review]

**Comment by lb-sgent** [comment]
image
Conflicts resolved

**Review by pk-sgent — APPROVED** [review]

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**